src/rust/ide/lib/view/src/graph_editor/component/edge.rs

```diff
@@ -513,11 +1253,33 @@ impl EdgeModelData {
 
         let fg               = &self.front;
         let bg               = &self.back;
-        let target_attached  = self.target_attached.get();
-        let node_half_width  = self.source_width.get() / 2.0;
-        let node_half_height = self.source_height.get() / 2.0;
-        let node_circle      = Vector2::new(node_half_width-node_half_height,0.0);
-        let node_radius      = node_half_height;
+        // FIXME This should be enabled for #672
+        // let fully_attached   = self.target_attached.get() && self.source_attached.get();
+        let fully_attached          = self.target_attached.get();
+        let node_half_width         = self.source_width.get() / 2.0;
+        let target_node_half_height = NODE_HEIGHT / 2.0;
+        let source_node_half_height = self.source_height.get() / 2.0;
+        let source_node_circle      = Vector2(node_half_width- source_node_half_height, 0.0);
+        let source_node_radius      = source_node_half_height;
+
+        // === Update Highlights ===
+
```

Comment:
such things should have 2 enters above and one below according to our style guide